protolite-well-known-types/proguard-rules.pro

```diff
@@ -19,3 +19,7 @@
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
 #-renamesourcefileattribute SourceFile
+
+-keepclassmembers class * extends com.google.protobuf.GeneratedMessageLite {
```

Comment:
Can you add a comment on why this is needed? It seems like we did not previously had to preserve these classes

Reply:
Done.